### ISSUE TITLE
relation between caffemodel and caffe

### ISSUE THREAD

**Issue by xuyunlu1030** [samiksome/C3D-weights#3]
Thanks for your source!
I meet with a confusing problem.

I used your provided v1.1 weights and model for finetuning on the UCF101 dataset, but the model can not converge with the pretrained model (loss is over 4)! (I also find: It can converge from the scratch.) It is really strange and I wonder why.
I guess any int model is not worse than a random initialization. So what's wrong with my training? is it some relationship between the caffemodel and caffe version? 

Could you give me some ideas? Thanks!

**Comment by xuyunlu1030** [comment]
I found the problem sulotion. Close the cudnn, it would be OK since the difference of the cudnn version.
